src/css/override_components.scss

```diff
@@ -135,15 +135,20 @@ button,
   }
 } /* stylelint-enable */
 
+.usa-button-disabled {
```

Comment:
taking the `usa-button-disabled` class off of the button the classes are `action action-primary` so the button looks active.

I could swap out the `usa-button-disabled` for `action-cautious` and that gets the same effect. 

As a side note, there a mix of styles that are a Frankenstein mix of `usa-` styles and cloud.gov styles.